website/docs/strong.md

```diff
@@ -0,0 +1,134 @@
+---
+id: strong
+title: Banning untyped from a file
+sidebar_label: Banning untyped
+---
+
+> **Note**: This feature is in beta. Please give us feedback!
+
+The `# typed: strong` sigil bans all usage of `T.untyped` in a file.
+
+![](/img/strong.png)
+
+Writing code at this level can be extremely exacting. We do not recommend using
+this level too early in the life cycle of adopting Sorbet, as it can cause an
+unreasonably high barrier to entry when changing the codebase in the future.
+Frequently, the best approach to dealing with `# typed: strong`-level errors is
+to simply ignore them, by downgrading the sigil to `# typed: strict`.
+
+For more information on our recommendations when adopting Sorbet, see
+[Suggestions for driving adoption](metrics.md#suggestions-for-driving-adoption).
+
+That being said, we welcome enterprising Sorbet users to try writing code at
+this level. If you do have feedback for where it's difficult or impossible to
+use this level, please let us know.
+
+## What counts as a usage of untyped?
+
+Some examples of things that count as a usage of untyped:
+
+- Calling a method on an untyped value
+- Passing an untyped value to a method
+- Conditionally branching on an untyped value
+- Returning an untyped value from a method
+- etc.
+
+The list of things that count as usages of untyped will grow over time (which is
+part of the reason why this feature is considered "beta").
+
+### What doesn't count?
+
+It's maybe more interesting to see what **doesn't** count as a usage of untyped:
+
+- `x = T.let(..., SomeType)` where `...` is an untyped expression.
+
+  This is one of the primary ways to fix errors arising in `# typed: strong`
+  files. The [`T.let`](type-assertions.md) type assertion ascribes a static type
+  to `x`. Using `x` in place of `...` means that Sorbet will see a usage of
+  something typed instead of something untyped.
+
+- Any usage of a type like `T::Array[T.untyped]` or
+  `T::Hash[String, T.untyped]`, which has `T.untyped` somewhere inside it.
+
+  We may reconsider this decision in the future. For now, Sorbet will at least
+  report code like this as using an untyped value:
```

Comment:
This section is about what *doesn't* count as untyped code, and yet this sentence says that Sorbet considers these things as not-untyped code?  Shouldn't the last sentence be inverted?

Reply:
It's trying to show that the usage of the _element_ is untyped, not that the usage of the array containing untyped is untyped. I can make that more explicit.